hazelcast/src/main/java/com/hazelcast/internal/util/phonehome/ClusterInfoCollector.java

```diff
@@ -37,7 +37,7 @@ public void forEachMetric(Node node, BiConsumer<PhoneHomeMetrics, String> metric
 
         metricsConsumer.accept(PhoneHomeMetrics.UUID_OF_CLUSTER, node.getThisUuid().toString());
         metricsConsumer.accept(PhoneHomeMetrics.CLUSTER_ID, clusterService.getClusterId().toString());
-        metricsConsumer.accept(PhoneHomeMetrics.CLUSTER_SIZE, MetricsCollector.convertToLetter(clusterSize));
```

Comment:
Please confirm with @AyberkSorgun that it is OK to have both letters and stringified numbers in the same column.
There was a suggestion to have 2 separate columns with the letter and integer types.